packages/terra-alert/src/Alert.scss

```diff
@@ -0,0 +1,141 @@
+@import './variables';
+
+:local {
+  /* stylelint-disable selector-class-pattern  */
+  /* stylelint-disable  property-no-unknown */
+  .alert-base {
+    align-items: stretch;
+    display: flex;
+    justify-content: space-between;
+    overflow: hidden;
+    width: 100%;
+  }
+
+  .narrow {
+    flex-direction: column;
+  }
+
+  .wide {
+    flex-direction: row;
+  }
+
+  .icon {
+    flex-shrink: 0;
+    font-size: var(--terra-alert-icon-font-size);
+    margin-right: 0.5rem;
+  }
+
+  .body {
+    align-items: flex-start;
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    line-height: 1.2;
+    margin-left: 0.6667rem;
+    margin-top: 0.7143rem;
+    overflow: hidden;
+    padding-top: 0;
+  }
+
+  .body-narrow {
+    composes: body;
+    margin-bottom: 0;
+    margin-right: 0.6667rem;
+  }
+
+  .body-std {
+    composes: body;
+    margin-bottom: 0.7143rem;
+    margin-right: 0;
+  }
+
+  .section {
+    flex: 1;
+    padding-bottom: 0.2143rem;
+    padding-top: 0.2143rem;
+  }
+
+  .section-custom {
+    color: var(--terra-alert-custom-text-color);
+    composes: section;
+  }
+
+  .title {
+    float: left;
+    margin-right: 0.5833rem;
+  }
+
+  .actions {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    justify-content: flex-end;
+    margin: 0.4286rem 0.5833rem 0.4286rem 0.8333rem;
+  }
+
+  .actions-custom {
+    color: var(--terra-alert-custom-text-color);
+    composes: actions;
+  }
+
+  .actions > :first-child {
+    margin-right: 0.8333rem;
```

Comment:
+1 on all previous style edits to fix icon/text/button alignment - they look great. 
Found one last edit:
`margin-right: 0.5714rem;`